lib/rules/web/admin_console/4.5/goto.xyaml

```diff
@@ -88,7 +89,7 @@ goto_deployment_page:
   url: k8s/ns/<project_name>/deployments
 goto_one_deployment_page:
   url: k8s/ns/<project_name>/deployments/<deploy_name>
-
+  action: wait_box_loaded
```

Comment:
Sometimes env is very slow to render the content, need a wait here just after goto. Refer to [the failed log](https://mastern-jenkins-csb-openshift-qe.cloud.paas.psi.redhat.com/job/Runner-v3/84838/console) without this rule added here. Failed in next step Click action.